### PR TITLE
Livetest fixes, readme and history

### PR DESCRIPTION
**Pull request by YijunXieMS**
#5110 Revise readme
#5083 Revise history
#5111 Multi python version
#5856 Fix missing consumer group directory in EPH
This PR mainly fixes live test code that doesn't work with Python 3.5 during python version compatibility test.
-------------------new pushes 17:30 on 06/17---------------
1. change create_receiver's event_position to be mandatory. This simple change caused a lot of changes in test and example code.
2. change EventPosition.first_available_event() to earliest() and new_events_only() to latest()

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/eventhub/azure-eventhubs/tests/asynctests/test_longrunning_send_async.py [hunk 1]
```diff
@@ -56,26 +52,16 @@ def check_send_successful(outcome, condition):
...
 
-    def data_generator():
-        for i in range(args.batch):
-            yield b"D" * args.payload
-
-    if args.batch > 1:
-        logger.info("{}: Sending batched messages".format(pid))
-    else:
-        logger.info("{}: Sending single messages".format(pid))
-
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/tests/asynctests/test_longrunning_send_async.py]: Why was this removed?
Without message batching this performance test will never get the needed throughput.
Batched messages should be the default operation of this test

File: sdk/eventhub/azure-eventhubs/setup.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 if not version:
     raise RuntimeError('Cannot find version information')
 
-with open('README.rst') as f:
+with open('README.md') as f:
     readme = f.read()
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/setup.py]: If the readme format has changed you will need to change the history.rst as well.
Also you will need to add the content type as markdown to the setup definition.

File: sdk/eventhub/azure-eventhubs/tests/asynctests/test_send_async.py [hunk 3]
```diff
@@ -242,3 +243,6 @@ def batched():
         received.extend(r.receive(timeout=3))
 
     assert len(received) == 20
+
+    for r in receivers:
+        r.close()
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/tests/asynctests/test_send_async.py]: This should be awaited?

File: sdk/eventhub/azure-eventhubs/tests/asynctests/test_receive_async.py [hunk 4]
```diff
@@ -309,14 +305,15 @@ def batched():
     for i in range(20):
         event_list.append(EventData("Event Number {}".format(i)))
 
-    async with receiver:
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/tests/asynctests/test_receive_async.py]: Why was this removed?
If we don't have the `async with` then if an error occurs during the test then the receiver will not be closed. Which can cause issues in subsequent tests.

File: sdk/eventhub/azure-eventhubs/tests/asynctests/test_send_async.py (hunk 3 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/tests/asynctests/test_send_async.py]: receivers are from conftest. They're not async. The sender is async in this test method

File: sdk/eventhub/azure-eventhubs/tests/asynctests/test_receive_async.py (hunk 4 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/tests/asynctests/test_receive_async.py]: Right. Changed back to use "async with".

File: sdk/eventhub/azure-eventhubs/setup.py (hunk 2 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/setup.py]: Changed history.rst to .md
